### ISSUE TITLE
fluid_synth_set_chorus_speed not bound?

### ISSUE THREAD

**Issue by joelcarlton** [nwhitehead/pyfluidsynth#51]
I'm trying to set some chorus values and I am getting errors that set_chorus_speed access fluid_synth_set_chorus_speed which is None.

**Comment by cclauss** [comment]
Agreed.  See our GitHub Actions.

% `ruff check --output-format=github --target-version=py38 .`
```
Error: fluidsynth.py:772:75: F821 Undefined name `chan`
Error: fluidsynth.py:888:12: F821 Undefined name `fluid_synth_set_chorus_speed`
Error: fluidsynth.py:889:20: F821 Undefined name `fluid_synth_set_chorus_speed`
Error: fluidsynth.py:893:12: F821 Undefined name `fluid_synth_set_chorus_depth`
Error: fluidsynth.py:894:20: F821 Undefined name `fluid_synth_set_chorus_depth`
Error: fluidsynth.py:1002:64: F821 Undefined name `name`
Error: fluidsynth.py:1002:83: F821 Undefined name `name`
Error: Process completed with exit code 1.
```
@nwhitehead @ChristianRomberg Are you able to fix undefined name `fluid_synth_set_chorus_speed`?

**Comment by cclauss** [comment]
@nwhitehead Can you please scroll to the lower right of this page and click `Pin issue` to raise this issue's visibility because it is causing our tests to fail?  Thanks for all your support.

**Comment by nwhitehead** [comment]
Should be fixed by #63.

@joelcarlton Could you give the current code in this repo a try to see if it fixes your issue?

**Comment by nwhitehead** [comment]
Closing as inactive, feel free to re-open if you think there is still an issue.
